frontend/src/store/modules/map/getters.js

```diff
@@ -4,5 +4,12 @@ export default {
     },
     getMapboxToken: function() {
         return process.env.MAPBOX_TOKEN;
+    },
+    getMapboxCenter: () => (places) => {
```

Comment:
Probably better send not all places from a store, do it more flexible. Maybe better array of objects here, because getMapboxCenter has to be a clean function and can't depend on store of vuex. Or, you can do like this

getMapboxCenter(placeHash, placeMap) { ... }

and after call it like this

getMapboxCenter(places.byId, places.allIds)